utils/ccloud_library.sh

```diff
@@ -908,7 +908,7 @@ function ccloud::create_ccloud_stack() {
   REPLICATION_FACTOR=${REPLICATION_FACTOR:-3}
   enable_ksqldb=${1:-false}
   EXAMPLE=${EXAMPLE:-ccloud-stack-function}
-  CHECK_CREDIT_CARD="${CHECK_CREDIT_CARD:-true}"
+  CHECK_CREDIT_CARD="${CHECK_CREDIT_CARD:-false}"
```

Comment:
What happens if user runs out of credits and there is no credit card on file?  Need to confirm how/where will this script fail, and will it fail fast.